test/PantosHub.t.sol

```diff
@@ -1107,46 +1272,92 @@ contract PantosHubTest is PantosHubDeployer {
 
-        registerToken(PANDAS_TOKEN_ADDRESS_2);
+        registerToken(PANDAS_TOKEN_ADDRESS_2, PANDAS_TOKEN_OWNER);
         tokenRegistered[2] = true;
         checkTokenRegistrations(tokenAddresses, tokenRegistered);
 
-        vm.prank(DEPLOYER);
+        vm.prank(PANDAS_TOKEN_OWNER);
         pantosHubProxy.unregisterToken(PANDAS_TOKEN_ADDRESS_1);
         tokenRegistered[1] = false;
         checkTokenRegistrations(tokenAddresses, tokenRegistered);
 
-        registerToken(PANDAS_TOKEN_ADDRESS_1);
+        registerToken(PANDAS_TOKEN_ADDRESS_1, PANDAS_TOKEN_OWNER);
         tokenRegistered[1] = true;
         checkTokenRegistrations(tokenAddresses, tokenRegistered);
 
-        vm.prank(DEPLOYER);
+        vm.prank(PANDAS_TOKEN_OWNER);
         pantosHubProxy.unregisterToken(PANDAS_TOKEN_ADDRESS);
         tokenRegistered[0] = false;
         checkTokenRegistrations(tokenAddresses, tokenRegistered);
 
-        vm.prank(DEPLOYER);
+        vm.prank(PANDAS_TOKEN_OWNER);
         pantosHubProxy.unregisterToken(PANDAS_TOKEN_ADDRESS_2);
         tokenRegistered[2] = false;
         checkTokenRegistrations(tokenAddresses, tokenRegistered);
 
-        registerToken(PANDAS_TOKEN_ADDRESS);
+        registerToken(PANDAS_TOKEN_ADDRESS, PANDAS_TOKEN_OWNER);
         tokenRegistered[0] = true;
         checkTokenRegistrations(tokenAddresses, tokenRegistered);
     }
 
     function test_registerExternalToken() external {
         registerToken();
 
-        vm.prank(DEPLOYER);
+        vm.prank(PANDAS_TOKEN_OWNER);
+        pantosHubProxy.registerExternalToken(
+            PANDAS_TOKEN_ADDRESS,
+            uint256(otherBlockchain.blockchainId),
+            EXTERNAL_PANDAS_TOKEN_ADDRESS
+        );
+
+        PantosTypes.ExternalTokenRecord
+            memory externalTokenRecord = pantosHubProxy.getExternalTokenRecord(
+                PANDAS_TOKEN_ADDRESS,
+                uint256(otherBlockchain.blockchainId)
+            );
+        assertTrue(externalTokenRecord.active);
+        assertEq(
+            externalTokenRecord.externalToken,
+            EXTERNAL_PANDAS_TOKEN_ADDRESS
+        );
+    }
+
+    function test_registerExternalTokenBySuperCriticalOps() external {
+        registerTokenBySuperCriticalOps();
+
+        vm.prank(SUPER_CRITICAL_OPS);
+        pantosHubProxy.registerExternalToken(
+            PANDAS_TOKEN_ADDRESS,
+            uint256(otherBlockchain.blockchainId),
+            EXTERNAL_PANDAS_TOKEN_ADDRESS
+        );
+
+        PantosTypes.ExternalTokenRecord
+            memory externalTokenRecord = pantosHubProxy.getExternalTokenRecord(
+                PANDAS_TOKEN_ADDRESS,
+                uint256(otherBlockchain.blockchainId)
+            );
+        assertTrue(externalTokenRecord.active);
+        assertEq(
+            externalTokenRecord.externalToken,
+            EXTERNAL_PANDAS_TOKEN_ADDRESS
+        );
+    }
+
+    function test_registerExternalTokenBySuperCriticalOpsAndPaused() external {
```

Comment:
Our naming convention would be `test_registerExternalToken_BySuperCriticalOpsAndPaused`.